src/core/server/csp/csp_directives.test.ts

```diff
@@ -47,12 +47,11 @@ describe('CspDirectives', () => {
       directives.addDirectiveValue('style-src', 'strict-dynamic');
       directives.addDirectiveValue('style-src', 'report-sample');
       directives.addDirectiveValue('style-src', 'unsafe-inline');
-      directives.addDirectiveValue('style-src', 'unsafe-eval');
```

Comment:
Not sure why this was removed. The purpose of the test is to assert that special keywords are properly escaped, and unsafe-eval is still a keyword (and user can still add it via custom configuration if they want to)

Reply:
Good call, I'll add it back. I was going all places in the code where the `unsafe-eval` was present and removed most of them.

Reply:
Fixed 👍